php/commands/term.php

```diff
@@ -417,6 +417,36 @@ public function url( $args ) {
 		}
 	}
 
+	/**
+	 * Recount the term_taxonomy count for one or more taxonomies.
+	 *
```

Comment:
Want to add a secondary, longer description about situations where this would be necessary?